graylog2-server/src/main/java/org/graylog2/entitygroups/model/EntityType.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+package org.graylog2.entitygroups.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Locale;
+
+public enum EntityType {
+    @JsonProperty("assets")
+    ASSETS,
+    @JsonProperty("sigma_rules")
+    SIGMA_RULES;
```

Comment:
Since this is in open we may want to find a way to keep enterprise entity references out as much as possible. Might be okay to leave them as strings and then from the entity service classes we can use the corresponding `EntityDBService.COLLECTION_NAME` variable to consistently set/retrieve the values.

Reply:
Yeah good point. This was merely forward looking to how we'll handle mapping the different types to their respective content pack entity type handling. But I know we have other ways of doing that so I think we can get a away with just String type keys, and will dig in more once we get to the content pack piece.